Governance/introduction.md

```diff
@@ -10,14 +10,15 @@ If you intend to share your data with colleagues for simple purposes, such as sa
 ## Chapter index
 
 ### **[Creating climate data](create/create-intro.md)**
-Creating datasets that are fit for publication can be a daunting process, with many technical aspects to consider. This chapter will outline the ways to ensure that your data meets the broad [FAIR principles](concepts/fairer-principles.md) of data sharing, along with specific requrirements and recommendations for climate-related data that will aid in the use and reproducibility of your data by the Australian, and international, climate community.
+Creating datasets that are fit for publication can be a daunting process, with many technical aspects to consider. This section will outline the ways to ensure that your data meets the broad [FAIR principles](concepts/fairer-principles.md) of data sharing, along with specific requrirements and recommendations for climate-related data that will aid in the use and reproducibility of your data by the Australian, and international, climate community.
 
 ### **[Publishing climate data](publish/publish-intro.md)**
-Publishing data involves uploading them to an accessible respository, and having a persistent identifier (e.g. a [DOI](https://www.doi.org/)) attached in a similar manner to publishing a research paper. This data can then be used and references by other researchers, increasing the scientific impact your dataresearch-ready data can have. Additionally, many journals now require that data on which a paper is based be published in this manner. In this chapter, the various pathways to data publication are presented, with a focus on recommendations for those at Australian research institutions.
+Publishing data involves uploading them to an accessible respository, and having a persistent identifier (e.g. a [DOI](https://www.doi.org/)) attached in a similar manner to publishing a research paper. This data can then be used and references by other researchers, increasing the scientific impact your dataresearch-ready data can have. Additionally, many journals now require that data on which a paper is based be published in this manner. In this section, the various pathways to data publication are presented, with a focus on recommendations for those at Australian research institutions.

```

Comment:
- This data can then be used and reference**s**d
- replace "dataresearch-ready" with research-ready data?
- many journals now require that data on which a paper is based **on** be

Reply:
Agree with the first two, 3rd suggestion adds a redundant 'on', so I think not that one.